test/tree_sitter/node_test.rb

```diff
@@ -396,4 +396,138 @@ def mul(a, b)
     assert_equal [@child.parameters, nil, @child.name, nil], @child.fetch(:parameters, :fake, :name, :fake)
     assert_equal [@child.parameters, nil, @child.name, nil], @child.fetch(:parameters, :fake, :name, :whatever)
   end
+
+  describe 'sexpr' do
+    it 'should print a proper sexpr when vertical is nil' do
```

Comment:
```
it 'prints...'
```

Reply:
I agree with this formulation, but I would like to keep it coherent for the time being. I will go through all tests and fix their descriptions, 1 shot.